src/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -362,13 +362,7 @@ public partial class SignedXml
         public const string XmlDsigMinimalCanonicalizationUrl = "http://www.w3.org/2000/09/xmldsig#minimal";
         public const string XmlDsigNamespaceUrl = "http://www.w3.org/2000/09/xmldsig#";
         public const string XmlDsigRSASHA1Url = "http://www.w3.org/2000/09/xmldsig#rsa-sha1";
-        public const string XmlDsigRSASHA256Url = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256";
-        public const string XmlDsigRSASHA384Url = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha384";
-        public const string XmlDsigRSASHA512Url = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha512";
         public const string XmlDsigSHA1Url = "http://www.w3.org/2000/09/xmldsig#sha1";
-        public const string XmlDsigSHA256Url = "http://www.w3.org/2001/04/xmlenc#sha256";
-        public const string XmlDsigSHA384Url = "http://www.w3.org/2001/04/xmldsig-more#sha384";
-        public const string XmlDsigSHA512Url = "http://www.w3.org/2001/04/xmlenc#sha512";
```

Comment:
Yes we cannot expose new APIs in a netstandard2.0 ref which don't exist on a platform that maps Ns2.0 like net461. We could add these in a netcoreapp specific version if we felt they were important enough but I personally would just leave them out, if we plan to stick with the netstandard2.0 approach.